<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>voxelGridProperties</class>
 <widget class="QDialog" name="voxelGridProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voxel Grid Properties:</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="fName">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDisabled">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Disabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowVoxel">
     <property name="text">
      <string>Show Voxel</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbPos">
     <property name="title">
      <string>Position [m]</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="xPos">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>x:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>xP</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <double>-100000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="yPos">
        <property name="text">
         <string>y:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>yP</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="yP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <double>-10000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="zPos">
        <property name="text">
         <string>z:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>zP</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="zP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <double>-10000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbRes">
     <property name="title">
      <string>Grid resolution</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lResX">
        <property name="text">
         <string>x:</string>
        </property>
        <property name="buddy">
         <cstring>spResX</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spResX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lResY">
        <property name="text">
         <string>y:</string>
        </property>
        <property name="buddy">
         <cstring>spResY</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spResY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lResZ">
        <property name="text">
         <string>z:</string>
        </property>
        <property name="buddy">
         <cstring>spResZ</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spResZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>4096</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSize">
     <property name="title">
      <string>Voxel size [m]</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lSizeX">
        <property name="text">
         <string>x:</string>
        </property>
        <property name="buddy">
         <cstring>dspSizeX</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dspSizeX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lSizeY">
        <property name="text">
         <string>y:</string>
        </property>
        <property name="buddy">
         <cstring>dspSizeY</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dspSizeY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lSizeZ">
        <property name="text">
         <string>z:</string>
        </property>
        <property name="buddy">
         <cstring>dspSizeZ</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dspSizeZ">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>voxelGridProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>voxelGridProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
